plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1090,11 +1114,15 @@ void producer_plugin::plugin_initialize(const boost::program_options::variables_
       }
    }
 
-   my->_ro_thread_pool_size = options.at( "read-only-threads" ).as<uint16_t>();
+   if ( options.count( "read-only-threads" ) ) {
+      my->_ro_thread_pool_size = options.at( "read-only-threads" ).as<uint16_t>();
```

Comment:
Can someone type `--read-only-threads 0` on producer node, but we don't allow?

Reply:
Fixed.